packages/core/addon/models/delivery-rule.ts

```diff
@@ -64,6 +64,9 @@ export default class DeliveryRuleModel extends Model.extend(Validations) {
   @fragment('fragments/delivery-format', { polymorphic: true })
   format!: DeliveryFormatFragment;
 
+  @attr({ defaultValue: 'Email delivered pdf every week' })
```

Comment:
I was trying to make it dynamic based on the other values but because they might not be defined I left it at the default values. Do you have a suggestion on how to do a better default?